plugins/modules/aci_switch_spine_selector.py

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Bruno Calogero <brunocalogero@hotmail.com>
+# Adapted from aci_switch_leaf_selector
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "certified",
+}
+
+DOCUMENTATION = r"""
+---
+module: aci_switch_spine_selector
+short_description: Bind spine selectors to switch policy spine profiles (infra:SpineS, infra:NodeBlk, infra:RsSpineAccNodePGrep)
+description:
+- Bind spine selectors (with node block range and policy group) to switch policy spine profiles on Cisco ACI fabrics.
+options:
+  description:
+    description:
+    - The description to assign to the C(spine).
+    type: str
+  spine_profile:
+    description:
+    - Name of the Spine Profile to which we add a Selector.
+    type: str
+    aliases: [ spine_profile_name ]
+  spine:
+    description:
+    - Name of Spine Selector.
+    type: str
+    aliases: [ name, spine_name, spine_profile_spine_name, spine_selector_name ]
+  spine_node_blk:
+    description:
+    - Name of Node Block range to be added to Spine Selector of given Spine Profile.
+    type: str
+    aliases: [ spine_node_blk_name, node_blk_name ]
+  spine_node_blk_description:
+    description:
+    - The description to assign to the C(spine_node_blk)
+    type: str
+  from:
+    description:
+    - Start of Node Block range.
+    type: int
+    aliases: [ node_blk_range_from, from_range, range_from ]
+  to:
+    description:
+    - Start of Node Block range.
+    type: int
+    aliases: [ node_blk_range_to, to_range, range_to ]
+  policy_group:
+    description:
+    - Name of the Policy Group to be added to Spine Selector of given Spine Profile.
```

Comment:
The name of the Policy Group to be added to the Spine Selector of the given Spine Profile.

Reply:
Done.